nexus-sdk/src/main/java/io/nexusrpc/handler/OperationHandlerException.java

```diff
@@ -1,42 +1,25 @@
 package io.nexusrpc.handler;
 
-import io.nexusrpc.FailureInfo;
 import org.jspecify.annotations.Nullable;
 
 /** Thrown from a handler for any unexpected error. */
 public class OperationHandlerException extends RuntimeException {
   private final ErrorType errorType;
-  private final FailureInfo failureInfo;
 
   public OperationHandlerException(ErrorType errorType, String message) {
```

Comment:
@bergundy I noticed in the original Go PR we have a helper `HandlerErrorf` to construct a `HandlerError` with a string, but we didn't add the same for `UnsuccessfulOperationError`. Is there a reason for this?

Reply:
I want to add those eventually. I'm debating whether `NewFailedOperationError` should take a string instead of a full error though. I kinda regret accepting an error.

We can discuss this offline though.